### PR TITLE
Fit- commit adding rading equires for admin

### PR DESCRIPTION
**Pull request by thepatrickniyo**
### **What does this PR do**
It add a feature for admin to read enquiries.

### **Feature Description**
Admin is able to view all sent enquiries from any user who sends it
Admin can see the sender names, email and time when it was done

### **How to manually test this**
 - Got [Login](https://patrickniyo.netlify.app/html/singin.html)
 - Login as admin
 - Go to Enquiries
 -  All unready enquires are displayed on 

### **Expected output**
 Admin will see all unread enquiries

### **Screenshot**
image

### CODE REVIEW

File: scripts/app.js [hunk 1]
```diff
@@ -167,6 +167,7 @@ function retreiveBlogs(){
                 console.log(profile.data().Names)
```
Comment (Anguandia) [inline scripts/app.js:167]: Please do not log to console, except in development

File: scripts/app.js [hunk 2]
```diff
@@ -300,10 +301,82 @@ function displayBlog(){
...
+    .then(() => {
+       successMessage.innerHTML = "Blog updted"
+        setTimeout(()=>{
+            successToaster.classList.remove('active')
+        },3000)
+        retreiveBlogs()
+    })
+    .catch(function(error) {
+        // The document probably doesn't exist.
+       alert("Error updating document: ", error);
```
Comment (Anguandia) [inline scripts/app.js:340]: Get a better way of displaying user feedback than alerts

**Review by Anguandia — APPROVED** [review]
Good work, you can merge for the demo but later, remember to create another PR to action the feedback herein

File: scripts/app.js (hunk 1 above)
Comment (thepatrickniyo) [reply scripts/app.js:167]: Eeehh ok! I didn't know. I will kindly correct it

File: scripts/app.js (hunk 2 above)
Comment (thepatrickniyo) [reply scripts/app.js:340]: Noted

**Pull request merged**